modules/claims_api/lib/bgs_service/manage_representative_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative 'manage_representative_service/read_poa_request'
+
+module ClaimsApi
+  class ManageRepresentativeService < ClaimsApi::LocalBGS
+    def endpoint
+      'VDC/ManageRepresentativeService'
+    end
+
+    def namespaces
+      {
+        'data' => 'http://gov.va.vba.benefits.vdc/data'
```

Comment:
any thought on trying the additional namespace without the host here? did you ever give that a go?
so something along the lines of 
```
'data' => '/data'
```

and in the namespace build try something like
```
%(xmlns:#{k}="#{namespace}#{v}")
```

Reply:
I didn't try this yet, but me + this PR are running thin on idea space (too much to think about at once). I think we can immediately follow on this change.